modules/api/static/schema.yml

```diff
@@ -0,0 +1,1937 @@
+            type: string
+        - name: filename
+          in: path
+          description: The image filename
+          required: true
+          schema:
+            type: string
+        - name: headername
+          in: path
+          description: The requested header name
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+           description: The header value
+           content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  Meta:
+                    type: object
+                    properties:
+                      CandID: 
+                        type: string
+                      Visit:  
+                        type: string
+                      File:  
+                        type: string
+                      Header:  
+                        type: string
+                  Value:
+                    type: string
+  '/candidates/{candid}/{visit}/qc/imaging':
+    get:
+      tags:
+        - Visit
+      summary: Get the imaging qc values of a visit
+      parameters:
+        - name: candid
+          in: path
+          description: The candidate identifier
+          required: true
+          schema:
+            type: string
+        - name: visit
+          in: path
+          description: The visit label
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+           description: The images list
+           content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ImagingQc'
+    put:
+      tags:
+        - Visit
+      parameters:
+        - name: candid
+          in: path
+          description: The candidate identifier
+          required: true
+          schema:
+            type: string
+        - name: visit
+          in: path
+          description: The visit label
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ImagingQc'
+      responses:
+        '204':
+          description: Qc value updated
+          
+  '/candidates/{candid}/{visit}/dicoms':
+    get:
+      tags:
+        - Dicoms
+      summary: List all dicoms for that timepoint
+      parameters:
+        - name: candid
+          in: path
+          description: ID of the candidate
+          required: true
+          schema:
+            type: string
+        - name: visit
+          in: path
+          description: This timepoint visit label
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: An object containing the list of dicoms
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Dicoms'
+    post:
```

Comment:
I think from here I cannot test it since the DICOM upload and processing is in a separate PR. Right?